__tests__/acceptance/eslint-with-todo-formatter-test.ts

```diff
@@ -1238,4 +1251,52 @@ describe('eslint with todo formatter', function () {
       }
     });
   }
+
+  it('when given FORMAT_TODO_AS will output with that formatters format', async () => {
+    project.write({
```

Comment:
This is actually an async function, so needs to be awaited (the API changed when i added `@scalvert/bin-tester`, as it uses a newer version of `fixturify-project`).

```suggestion
    await project.write({
```